src/components/Book.js

```diff
@@ -0,0 +1,18 @@
+import './Book.css';
+import PropTypes from 'prop-types';
+
+const Book = ({ bookInfo }) => (
+  <div className="Book-container" key={bookInfo.id}>
+    <h3 className="book-title">{bookInfo.title}</h3>
+    <h3 className="book-author">{bookInfo.author}</h3>
+  </div>
+);
```

Comment:
- [ ] It would be very nice if you add a remove button here in the Book Component because in the future you would add functionality for the removing book so you would need a button 🤔 Kindly add a remove button here (Not Functional Now) to follow the project requirements 👍